tests/unit/s2n_client_key_share_extension_pq_test.c

```diff
@@ -174,24 +177,34 @@ int main()
                         EXPECT_SUCCESS(s2n_stuffer_read_uint16(&key_share_extension, &sent_hybrid_iana_id));
                         EXPECT_EQUAL(sent_hybrid_iana_id, kem_pref->tls13_kem_groups[0]->iana_id);
 
-                        uint16_t expected_hybrid_share_size =
-                                S2N_SIZE_OF_KEY_SHARE_SIZE
-                                + test_kem_group->curve->share_size
-                                + S2N_SIZE_OF_KEY_SHARE_SIZE
-                                + test_kem_group->kem->public_key_length;
+                        uint16_t expected_hybrid_share_size;
+
+                        if (len_prefixed) {
+                            expected_hybrid_share_size = S2N_SIZE_OF_KEY_SHARE_SIZE
+                                    + test_kem_group->curve->share_size
+                                    + S2N_SIZE_OF_KEY_SHARE_SIZE
+                                    + test_kem_group->kem->public_key_length;
+                        } else {
+                            expected_hybrid_share_size = test_kem_group->curve->share_size + test_kem_group->kem->public_key_length;
+                        }
+
                         uint16_t sent_hybrid_share_size;
                         EXPECT_SUCCESS(s2n_stuffer_read_uint16(&key_share_extension, &sent_hybrid_share_size));
                         EXPECT_EQUAL(sent_hybrid_share_size, expected_hybrid_share_size);
 
-                        uint16_t hybrid_ecc_share_size;
-                        EXPECT_SUCCESS(s2n_stuffer_read_uint16(&key_share_extension, &hybrid_ecc_share_size));
-                        EXPECT_EQUAL(hybrid_ecc_share_size, test_kem_group->curve->share_size);
-                        EXPECT_SUCCESS(s2n_stuffer_skip_read(&key_share_extension, hybrid_ecc_share_size));
+                        if (len_prefixed) {
+                            uint16_t hybrid_ecc_share_size;
+                            EXPECT_SUCCESS(s2n_stuffer_read_uint16(&key_share_extension, &hybrid_ecc_share_size));
```

Comment:
Nit: initialize all variables. Even where a later line or another function initializes the variable before it's used, it can make identifying the actually uninitialized variables harder when we invariably get a test randomly failing on a very old compiler.

Can you fix the other instances too? There's several more in the tests, and also a couple in the actual source.
```suggestion
                            uint16_t hybrid_ecc_share_size = 0;
                            EXPECT_SUCCESS(s2n_stuffer_read_uint16(&key_share_extension, &hybrid_ecc_share_size));
```